feat(tianmu): Improve the readbility of stonedb. (#11)

konghaiya opened the pull request:
[summary]
    1 class member variable refactor：exproter directory;
    2 class member variable refactor：loader directory;

<!--

Thank you for contributing to StoneDB!

PR Title Format: <type>(<scope>): description to this pr (#issue_id)
e.g.
fix(util): fix sth..... (#1)

-->

## Summary about this PR
<!--

There MUST be one line starting with "Issue Number:  " and 
linking the relevant issues via the "close" or "ref".
e.g.:
Issue: close #1

-->

Issue Number: close #issue_number_you_created


## Tests Check List
<!-- At least one of next options must be included. -->

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

## Changelog
<!-- At least one of next options must be included. -->

- [ ] New Feature
- [ ] Bug Fix
- [x] Improvement
- [ ] Performance Improvement
- [ ] Build/Testing/CI/CD
- [ ] Documentation
- [ ] Not for changelog (changelog entry is not required)

## Documentation
<!-- At least one of next options must be included. -->

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features

hustjieke reviewed (approved):
LGTM

RingsC commented on storage/tianmu/exporter/data_exporter.h:
```diff
@@ -46,22 +46,22 @@ class DataExporter {
-  std::vector<core::AttributeTypeInfo> deas;
-  std::vector<core::AttributeTypeInfo> source_deas;
-  fields_t _fields;
+  int cur_attr_;
```
You should keep in mind that: 
readability is not just adding some underscore, but let other to know what you want to do just only via your code.

Therefore,` deas_` means what ? I think `attr_infos_` or something else is a better choice.

konghaiya replied:
Some obscure names have been adjusted

RingsC commented on storage/tianmu/exporter/data_exporter.h:
```diff
@@ -46,22 +46,22 @@ class DataExporter {
-  std::shared_ptr<system::LargeBuffer> buf;
-  system::ChannelOut *progressout;
+  std::shared_ptr<system::LargeBuffer> buf_;
+  system::ChannelOut *progressout_;
```
all same things.

konghaiya committed:
feat(tianmu): Improve the readbility of stonedb. (#11)
[summary]
    1 class member variable refactor：exproter directory;
    2 class member variable refactor：loader directory;

konghaiya committed:
Merge branch 'stonedb-5.7-dev' into dev_lhj_1101

mergify[bot] committed:
Merge branch 'stonedb-5.7-dev' into dev_lhj_1101
